firebaseoptions/app/src/main/java/devrel/firebase/google/com/firebaseoptions/MainActivity.java

```diff
@@ -30,22 +29,25 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         // [START firebase_options]
-        // Manually configure Firebase Options
+        // Manually configure Firebase Options. The following fields are REQUIRED:
+        //   - Project ID
+        //   - App ID
+        //   - API Key
         FirebaseOptions options = new FirebaseOptions.Builder()
-                .setApplicationId("1:27992087142:android:ce3b6448250083d1") // Required for Analytics.
-                .setApiKey("AIzaSyADUe90ULnQDuGShD9W23RDP0xmeDc6Mvw") // Required for Auth.
-                .setDatabaseUrl("https://myproject.firebaseio.com") // Required for RTDB.
+                .setProjectId("<PROJECT_ID>")
+                .setApplicationId("<APP_ID>")
+                .setApiKey("<API_KEY>")
+                // setDatabaseURL(...)
+                // setStorageBucket(...)
```

Comment:
I would recommend setting these too, to make it clear to developers that they should just always set the full set of values and not select for the task at hand.

Also I recommend to use example values (as it was done before).

Reply:
It's likely that Firebase moves to a place where not all of these resources are created on project creation.  

I don't think we should do any more silent breaking changes in the future.  If we want to require DatabaseURL tomorrow we should break FirebaseApp.

Reply:
(changed back to example values)